### PR TITLE
Some fixes for doxygen syntax errors and some configuration to make it…

### PR DESCRIPTION
**Pull request by 7ofNine**
… run smoother under Windows.

### CODE REVIEW

File: src/Mod/PartDesign/Gui/Utils.h [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #ifndef UTILS_H_CS5LK2ZQ
 #define UTILS_H_CS5LK2ZQ
 
-/** \file Utils.h
+/** \file PartDesign/Gui/Utils.h
  *  This fiel contains some utility function used over PartDesignGui module
```
Comment (luzpaz) [inline src/Mod/PartDesign/Gui/Utils.h]: typo: should be `file`

File: src/Mod/Spreadsheet/App/Cell.cpp [hunk 2]
```diff
@@ -72,10 +72,8 @@ const int Cell::ALIGNMENT_VERTICAL   = 0xf0;
 
 /**
   * Construct a CellContent object.
-  *
-  * @param _row   The row of the cell in the spreadsheet that contains is.
-  * @param _col   The column of the cell in the spreadsheet that contains is.
-  * @param _owner The spreadsheet that owns this cell.
+  * @param _address  The adress of the caell
```
Comment (luzpaz) [inline src/Mod/Spreadsheet/App/Cell.cpp]: `The address of the cell`

Comment (luzpaz) [inline src/Mod/Spreadsheet/App/Cell.cpp]: Also what happened to:
`@param _row` and `@param _col` ?

File: src/App/Range.cpp [hunk 3]
```diff
@@ -197,9 +197,7 @@ int App::validColumn(const std::string &colstr)
   *
   * This function will throw an exception if the specified \a address is invalid.
   *
-  * @param address Address to parse.
-  * @param row     Reference to integer where row position is stored.
-  * @param col     Reference to integer where col position is stored.
+  * @param strAddress Address to parse.
   *
```
Comment (luzpaz) [inline src/App/Range.cpp]: `@param col` and `@param row` ?
same as https://github.com/FreeCAD/FreeCAD/pull/564/files#r103235811

File: src/Mod/Spreadsheet/App/Cell.cpp (hunk 2 above)
Comment (7ofNine) [inline src/Mod/Spreadsheet/App/Cell.cpp]: Oops, should check my own typos, too

File: src/Mod/PartDesign/Gui/Utils.h (hunk 1 above)
Comment (7ofNine) [inline src/Mod/PartDesign/Gui/Utils.h]: It's a reference to a file i.e. it should be \file but doxygen complains about ambiguity. There are two files in its realm that are called Utils.h and it wants to know specifically which one is referenced.

File: src/Mod/Spreadsheet/App/Cell.cpp (hunk 2 above)
Comment (7ofNine) [inline src/Mod/Spreadsheet/App/Cell.cpp]: They have been removed from the method(or where never there). Not by me. The comments are just reflecting what's existing to shut-up doxygen.

File: src/App/Range.cpp (hunk 3 above)
Comment (7ofNine) [inline src/App/Range.cpp]: That's what doxygen complains about these parameters are nowhere to be seen in the actual declaration/definition and address has been renamed to strAddress. The code hasn't been touched. The doxygen comments are just reflecting what's implemented.

File: src/Mod/PartDesign/Gui/Utils.h (hunk 1 above)
Comment (luzpaz) [inline src/Mod/PartDesign/Gui/Utils.h]: Sorry, not sure why the github UI is not showing it more prominently but i was referring to line 27
https://github.com/FreeCAD/FreeCAD/pull/564#discussion-diff-103234538L27

Comment (7ofNine) [inline src/Mod/PartDesign/Gui/Utils.h]: I didn't see it either. A historic typo. I'll fix it

File: src/Gui/Quarter/Quarter.cpp [hunk 4]
```diff
@@ -121,22 +121,23 @@
...
-
-  \subpage dynamicui
-
-  \subpage inheritui
-
-  \subpage mdi
-
-  \subpage examiner
-*/
+// These pages are not exitant in this copy of Quarter
```
Comment (luzpaz) [inline src/Gui/Quarter/Quarter.cpp]: `These pages are not existant in this copy of Quarter`
or
`These pages do not exist in this copy of Quarter`
Think the 2nd one makes more sense. 
Question though: why does it say immediately below that 
```
//  The examples code is included in Quarter and can be found in the
```

Comment (7ofNine) [inline src/Gui/Quarter/Quarter.cpp]: Well I am open to suggestions.
I think the needed part of a version of Quarter was transferred into the FreeCad source tree but not the examples which is referred to in the doxygen comments. But I didn't want to remove the original text. The added comment could be somewhat more explanatory. It was a quick shot when I tried to make the SourceDocu run "properly" (i.e. better than before, still some errors) in order to actually get to the documentation

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**